activerecord/test/models/book.rb

```diff
@@ -1,5 +1,5 @@
 class Book < ActiveRecord::Base
-  has_many :authors
```

Comment:
The schema has the `_id` column on the `books` table, not `authors`, https://github.com/rails/rails/blob/master/activerecord/test/schema/schema.rb#L55-L61 versus https://github.com/rails/rails/blob/master/activerecord/test/schema/schema.rb#L91